assets/js/params/File/UploadedFileSelect.js

```diff
@@ -105,11 +107,15 @@ const UploadedFileSelect = React.memo(function UploadedFileSelect ({ value, file
 
   const valueIndex = files.findIndex(({ uuid }) => uuid === value)
   const canOpen = files.findIndex(({ uuid }) => uuid !== value) !== -1 // do not open when nothing to select
+  const nFiles = files.length
+  const fileNumber = valueIndex === -1 ? <Trans id='js.params.File.UploadedFileSelect.choosePreviousFile.none'>[NONE]</Trans> : (valueIndex + 1)
```

Comment:
Sigh, I just realized that `valueIndex + 1` won't be number-formatted when we use this pattern.

I say, ignore the bug. It may not ever affect anybody, since the numbers are always small.